src/pages/registration/description_tab/projects_field/ProjectContributorRow.tsx

```diff
@@ -100,9 +100,8 @@ export const ProjectContributorRow = ({
     setIsLoadingDefaultOptions(false);
   };
 
-  const contributorErrors = errors.contributors?.[contributorIndex] as FormikErrors<ProjectContributor>;
-
-  const isRekProjectManager = isRekProject && contributor?.type === 'ProjectManager';
+  const contributorErrors = errors?.contributors?.[contributorIndex] as ProjectContributor;
```

Comment:
burde ikke denne fortsatt wrappes i FormikErrors?
```suggestion
  const contributorErrors = errors?.contributors?.[contributorIndex] as FormikErrors<ProjectContributor>;
```

Reply:
Jeg fikk feil der den ikke ville anerkjenne type-oppbyggingen når jeg wrappet den i FormikErrors. Og siden det funket uten tenkte jeg at det var greit. Vi kan ta en prat om det om du vil gå mer innpå det, ellers gjelder vel det vi begge har skrevet over om at det er et snart utdatert view.